<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>framelessWidget</class>
 <widget class="QWidget" name="framelessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
}
QPushButton:hover {
    border-color: #3498db; /* 鼠标悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #d5eaf2; /* 按下按钮时的背景颜色，非常浅的蓝色 */
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QWidget" name="mainWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="mainLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="titleBar" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#titleBar{

background-color: rgb(251, 251, 251);}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_min">
                   <property name="minimumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#f9bf45;border-radius:9px;}
QPushButton:pressed {background-color:#ffb11b;border-radius:9px;}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_max">
                   <property name="minimumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#227d51;border-radius:9px;}
QPushButton:pressed {background-color:#2d6d4b;border-radius:9px;}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_close">
                   <property name="minimumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#cb4042;border-radius:9px;}
QPushButton:pressed {background-color:#ab3b3a;border-radius:9px;}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="mainDisplayWidget" native="true">
              <property name="styleSheet">
               <string notr="true">#mainDisplayWidget{
background-color: rgb(251, 251, 251);}</string>
              </property>
              <widget class="QPushButton" name="pushButton">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>30</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	image: url(:/resource/image/user.png);
    border-radius: 25px; /* 设置按钮为圆形，半径为按钮宽度的一半 */
    width: 50px; /* 设置按钮的宽度 */
    height: 50px; /* 设置按钮的高度 */

    color: white; /* 设置文字颜色为白色 */
    font-size: 16px; /* 设置文字大小 */
    border: none; /* 去除按钮的边框 */
}

QPushButton:hover {
    background-color: #d5eaf2; /* 按下按钮时的背景颜色，非常浅的蓝色 */
}

QPushButton:pressed {
    background-color: #3498db; /* 按下按钮时的背景颜色，更深的蓝色 */
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_2">
               <property name="geometry">
                <rect>
                 <x>780</x>
                 <y>700</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/resource/image/pre.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_3">
               <property name="geometry">
                <rect>
                 <x>890</x>
                 <y>700</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/resource/image/pause.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_4">
               <property name="geometry">
                <rect>
                 <x>1000</x>
                 <y>700</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/resource/image/next.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_5">
               <property name="geometry">
                <rect>
                 <x>650</x>
                 <y>700</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/resource/image/dir.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_6">
               <property name="geometry">
                <rect>
                 <x>1110</x>
                 <y>700</y>
                 <width>50</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/resource/image/next.png);
image: url(:/resource/image/volume.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
